### PR TITLE
Update Kiosk-Browser + fix HB

### PR DESCRIPTION
**Pull request by malex984**
Add Kiosk Browser 0.9.8 with `preload` feature and corresponding wrapper script 

* cleanup due to separation of Kiosk-Browser into its own repository
* update of Kiosk-Browser wrappers

### CODE REVIEW

**Review by porst17 — CHANGES_REQUESTED** [review]
Why copy all these files over? You always have to keep them in sync!

Why not use the `ADD` command in `Dockerfile` and link directly to a particular commit (since we didn't tag anything lately). But we could also make beta releases for that. This way we would finally be able to track at least minimal dependencies. Something I am pretty much requesting since the very beginning of the project.

**Comment by malex984** [comment]
This is temporary since new ADDed file are not yet part of HB repository (the changes are due to https://github.com/hilbert/hilbert-heartbeat/pull/19/commits/95aab1104226b069db02a1d7eaba55efa0c9f5c3). 
Once HB repository accepts PR it will be simple to ADD them properly

**Comment by malex984** [comment]
of course i can ADD files from temporary branch on HB repo. instead...?

**Comment by porst17** [comment]
Even if the branch is temporary, the commit isn't. So you directly link to it.

Also: Is is really necessary to add temporary changes to this repo? Why not just wait until the heartbeat PR is approved, i.e. changes are kind of final. Otherwise you will have to re-sync each time there is a change in that PR.

**Comment by malex984** [comment]
@porst17 Ok, i *removed* those fixes due to switch from [sec] to [ms] and fixed which version of `client/web/hilbert-heartbeat.js` to get from HB repo (the latest with your latest changes, merged into `devel`). 
ps: the rest of changes will have to be added after we fix HB repository wrt. that.

File: images/kiosk/Makefile [hunk 1]
```diff
@@ -2,24 +2,31 @@ mkfile_path := $(abspath $(lastword $(MAKEFILE_LIST)))
...
 # TAG=titan.h-its.org:5555/${IMG}
 # titan.h-its.org:5555/hits/moon:${IMAGE_VERSION:-latest}
 
-
+# --ipc=host
+# -v /dev/shm:/dev/shm:rw
+# --device /dev/dri/card1
 # --privileged
-RUNTERM=--rm -it -a stdin -a stdout -a stderr --ipc=host --net=host 
+RUNTERM=--rm -it -a stdin -a stdout -a stderr --net=host -v /tmp/:/tmp/:rw -e DISPLAY --device /dev/dri/card1 --device /dev/dri/renderD129
```
Comment (porst17) [inline images/kiosk/Makefile]: Are you sure that devices will always be named that way?

**Review by porst17 — DISMISSED** [review]
Generally OK, but please check my question about mounting `/dev/dri/` devices.

File: images/kiosk/Makefile (hunk 1 above)
Comment (malex984) [reply images/kiosk/Makefile]: no, not always. I will remove those devices - they are just to try to get GPU support while checking Kiosk-Browser

Comment (porst17) [reply images/kiosk/Makefile]: Would it be more robust if you would mount *all* devices in `/dev/dri/`?

**Comment by malex984** [comment]
ok, now i look for any character device in `/dev/`, which belongs to `video` group

File: images/kiosk/Makefile [hunk 2]
```diff
@@ -2,24 +2,36 @@ mkfile_path := $(abspath $(lastword $(MAKEFILE_LIST)))
...
 ARGS=
-APP_ID=browser_app
 
-#APP=kiosktest
+# APP=kiosktest
 # IMG  = ${U}/${APP}:${IMAGE_VERSION}
 # TAG=titan.h-its.org:5555/${IMG}
 # titan.h-its.org:5555/hits/moon:${IMAGE_VERSION:-latest}
 
+VIDEO_DEVICES=`find /dev/ -group video -type c -print 2>/dev/null | xargs -L 1 -n 1 --no-run-if-empty -I '{}' echo "--device {}" | xargs --no-run-if-empty`
```
Comment (porst17) [inline images/kiosk/Makefile]: Would this be equivalent to `find /dev/ -group video -type c -exec echo -n "--device {} " \;` or am I missing something important?

Comment (malex984) [reply images/kiosk/Makefile]: No, it does the same thing. and even shorter and better IMO. Thanks for the hint!

**Review by porst17 — APPROVED** [review]

**Pull request merged**